packages/evolution-frontend/tests/ui-testing/testHelpers.ts

```diff
@@ -458,23 +458,29 @@ export const inputCheckboxTest: InputCheckboxTest = ({ context, path, values })
 };
 
 // Test input mapFindPlace widget
-export const inputMapFindPlaceTest: InputMapFindPlaceTest = ({ context, path }) => {
+export const inputMapFindPlaceTest: InputMapFindPlaceTest = ({ context, path, expectMultiple = true }) => {
```

Comment:
Great idea! I did try to handle this bug, but I didn't think about adding the expectMultiple parameter.

Reply:
Ce qui est plate, c'est que ça dépend d'un API externe (google) qui retourne un nombre assez inconnu de résultat. ça se pourrait qu'un jour ça retourne un nombre différent. Dans un monde idéal, il faudrait catcher ces requêtes et mocker la réponse, je vais ouvrir un issue pour ça